src/Datadog/Datadog/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+## Version 0.2.0
```

Comment:
Please remove '## Version 0.2.0'. New changelog should be put under 'Upcoming Release'